src/Test/Utilities/Portable/Compilation/TestOperationWalker.cs

```diff
@@ -537,5 +515,22 @@ public override void VisitInvalidExpression(IInvalidExpression operation)
         {
             base.VisitInvalidExpression(operation);
         }
+
+        private void VisitArgumentsInParameterOrder(IHasArgumentsExpression operation, ImmutableArray<IParameterSymbol>? parameters)
```

Comment:
>VisitArgumentsInParameterOrder [](start = 21, length = 30)

I am not sure what this method is doing. Isn't it visiting some arguments twice?